src/app/main/listing/droid-listing/assassin-droid/assassin-droid.component.html

```diff
@@ -1,4 +1,4 @@
-<fieldset [formGroup]="formGroup" class="container">
+<fieldset *ngIf="formGroup" [formGroup]="formGroup" class="container">
```

Comment:
Annoying check but has to be done...

1) we create the form from the abstract class based on an attribute of the child class and the assignation cannot be done within the `constructor` because of that, it's done within `ngOnInit`

2) because of the memory leak, we might destroy the form so strict check with undefined is also working into the view

When the angular issue is solved we might try to figure out something better, or eventually just ask devs using the lib to create the form themselves :( ...